NSS-Spontaneous-Spoonbills/Sprint2
# Dp department details

## Link to Ticket

#7 

## Description of Proposed Changes

## Steps to Test

Outline the steps to test

cd to ../sprint2
```
git fetch --all
git checkout dp_department_details
rm -rf db.sqlite3
cd WorkforceManagement/migrations
ls
```
* nuke everything in migrations besides init.py and pycache
```
../../
python manage.py makemigrations WorkforceManagement
python manage.py migrate
python manage.py runserver
```
1. open db and add some entries into the departments table, and the employees table, and all the dependancies and like, whatever man.... 
1. navigate to http://localhost:8000/WorkforceManagement/departments/ and see if the departments are listed, clickable and when you do click it displays the details of the department (budget and employees)

## Impacted Areas in Application
nav, departments, employees, computers

## Mentions @username
@MrErin @JacobAndrewSmith92 @kimberly-bird @binauralbeat 

## Testing




## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
